### ISSUE TITLE
App's touchpoints modal does not maintain focus in the modal

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#8573]
## Area or Component

App - Touchpoints modal

## Quick Summary
When opening the touchpoints modal in the app, focus is not maintained in the modal. Tab through with a keyboard and focus leaves the modal.

### How to Replicate
- Open the app
- Tab through modal contents and see that the focus leaves the modal (compare this with other modals in our application)

## Accessibility Violation

[reading-order-browse-outside-modal](https://docs.deque.com/issue-help/1.0.0/en/reading-order-browse-outside-modal)
[focus-modal-moves-outside
](https://docs.deque.com/issue-help/1.0.0/en/focus-modal-moves-outside)
## Action Requested

Ensure focus is maintained within the touchpoints modal

**Comment by emyl3** [comment]
⚠️ Added the other Touchpoints modal issue as a sub issue because I believe it makes sense to fix all of these together
